tools/docker/Dockerfile

```diff
@@ -1,12 +1,11 @@
-FROM alpine:3.5
-
-MAINTAINER Mark Harrison <mark@mivok.net>
-
-RUN apk add --no-cache ruby && \
-    gem install --no-rdoc --no-ri mdl && \
+FROM alpine:3.11.3
+RUN echo "mdl:x:1000:mdl" >> /etc/group && \
+    echo "mdl:x:1000:1000:mdl,,,:/home/mdl:/sbin/nologin" >> /etc/passwd && \
+    apk add --update --no-cache \
```

Comment:
I just noticed, you're missing an `&&` at the end of this line.

Reply:
It works. It is just a multi-line to improve readability.

Reply:
It is fine by me to put `ruby` on the same line.